components/accordion/index.tsx

```diff
@@ -0,0 +1,34 @@
+import React, {useState} from 'react';
+import Collapse, {IProps as CollapseProps} from '../collapse';
+
+interface IProps {
+  data: Array<CollapseProps>;
+}
+
+const Accordion = ({data}: IProps) => {
```

Comment:
You can use the approach with using official typings for functional component:
```suggestion
const Accordion: React.FC<IProps> = ({data}) => {
```